modules/claims_api/app/controllers/concerns/claims_api/dependent_claimant_verification.rb

```diff
@@ -0,0 +1,50 @@
+# frozen_string_literal: true
+
+require 'bgs_service/person_web_service'
+
+module ClaimsApi
+  module DependentClaimantVerification
+    extend ActiveSupport::Concern
+
+    included do
+      def validate_dependent_by_participant_id!(participant_id, dependent_first_name, dependent_last_name)
+        return if valid_participant_dependent_combo?(participant_id, dependent_first_name, dependent_last_name)
+
+        raise ::Common::Exceptions::InvalidFieldValue.new('participant_id: dependent combo',
+                                                          "#{participant_id}: #{dependent_first_name} " \
+                                                          "#{dependent_last_name}")
+      end
+    end
+
+    private
+
+    def normalize_name(name)
+      return '' unless name
+
+      name.strip.upcase
+    end
+
+    def valid_participant_dependent_combo?(participant_id, dependent_first_name, dependent_last_name)
+      return false if participant_id.blank?
+
+      person_web_service = PersonWebService.new(external_uid: 'dependent_claimant_verification_uid',
+                                                external_key: 'dependent_claimant_verification_key')
+      dependents = person_web_service.find_dependents_by_ptcpnt_id(participant_id)
+
+      return false if dependents.nil? || dependents[:number_of_records].to_i.zero?
+
+      Array.wrap(dependents).any? do |dependent|
```

Comment:
see: https://apidock.com/rails/Array/wrap/class

we know we [get an object response](https://github.com/department-of-veterans-affairs/vets-api/blob/ec36fe81aed483adb69a3b1f8a3cea666bd18542/modules/claims_api/spec/lib/claims_api/person_web_service_spec.rb) if there is one dependent. we don't know if we get an object or array if there are multiple dependents. wrap should handle both cases.

n.b. we assume each dependent will have the `dependent` key at the root level in the "many" case as it does in the "singular" case. might not be true.

Reply:
we found a participant with multiple dependents. turns out there is one `dependent` key that contains an array. thanks Rocky! https://github.com/department-of-veterans-affairs/vets-api/pull/18416/commits/19811d35469c941e60f7d8341d5e670d8095295d